lib/media/track/sender.js

```diff
@@ -26,7 +29,19 @@ class MediaTrackSender extends MediaTrackTransceiver {
    * @returns {MediaTrackSender}
    */
   clone() {
-    return new MediaTrackSender(this.track.clone());
+    const clone = new MediaTrackSender(this.track.clone());
+    this._clones.add(clone);
+    return clone;
+  }
+
+  replaceTrack(newTrack) {
+    this.track = newTrack;
+    this._senders.forEach(sender => {
+      if (sender.track) {
+        sender.replaceTrack(this.track).catch(() => {});
+      }
+    });
+    this._clones.forEach(clone => clone.replaceTrack(newTrack.clone()));
```

Comment:
@manjeshbhargav, I will update to return promise, however I am not very sure about not stopping the track if replaceTrack fails. Because I have seen replaceTrack failing often (because peerConnection being in closed state). I think its more like we need to remove sender when they are no more in use, but looks like its not happening.